JPS_VIRTUALSENSOR/JurongIslandInputAgent/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.8"
+
+services:
+  jurong-island-input-agent:
+    image: ghcr.io/cambridge-cares/jurong-island-input-agent:1.0.0-1557-develop-jurong-island-input-agent-SNAPSHOT
+    deploy:
+      mode: replicated
+      replicas: 1
+      restart_policy:
+        condition: none
+    environment:
+      - STACK_NAME=${STACK_NAME}
+      - DATABASE=postgres
```

Comment:
not needed?

Reply:
The STACK_NAME and DATABASE environment variables have been removed.

Reply:
STACK_NAME is required I believe? Sorry i was not clear, was just referring to DATABASE